### ISSUE TITLE
Less issue vs a question...

### ISSUE THREAD

**Issue by lethalox** [j-c-m/ubnt-letsencrypt#48]
Great Tool by the way worked like a champ.    My question is such,  I registered the subdomain as "edgerouter.foo.org", which worked.   But now I would like to register not only "edgerouter.foo.org" but also "edgerouter" & the IP of the router.   Can I do that?   If so, do I have to uninstall?  If not what commands to have clear out? 

Much thanks!

**Comment by marcovn** [comment]
I don't think you can include hostnames that are not publicly resolvable. So "edgeroute.foo.org" will work as it points to your external IP. But edgerouter does not. ACME does not allow IP addresses in a certificate (I think this is not good practice anyway).
